### PR TITLE
debug split() and -a option (print all)

### PR DESCRIPTION
**Pull request by hiroshi-matsuda**

### CODE REVIEW

File: sudachipy/command_line.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 
 def run(tokenizer, mode, reader, output, print_all):
-    for line in reader.readlines():
+    for line in iter(sys.stdin.readline, ""):
```
Comment (kazuma-t) [inline sudachipy/command_line.py]: This change breaks arguments of this tool. This tool reads files as input. If no files are specified, it reads STDIN.

Comment (kazuma-t) [reply sudachipy/command_line.py]: We can use `fileinput`

```python
import fileinput

for line in fileinput.input(args):
    print line
```

**Review by kazuma-t — CHANGES_REQUESTED** [review]

File: sudachipy/command_line.py (hunk 1 above)
Comment (hiroshi-matsuda) [reply sudachipy/command_line.py]: Oh, It's bad stile. I'll correct it soon.

**Comment by hiroshi-matsuda** [comment]
@kazuma-t Please check the last commit. I want to execute analysis immediately when new line comes from input stream.

**Review by kazuma-t — APPROVED** [review]

**Comment by kazuma-t** [comment]
Does this change (2f6861004fbc7a725f83e7b124570ca7fc8bbe6b) work well?

**Comment by hiroshi-matsuda** [comment]
I tested command_line.py with stdin and multiple input files and found no problems.

**Comment by hiroshi-matsuda** [comment]
Above comment is for my last commit.
I think 2f68610 line #13 is not exactly equivalent to java BufferedReader#readlLine().
I actually separate \n and \r to emulate java's behaviour.
Other lines are good for me.

**Comment by kazuma-t** [comment]
BufferedReader#readlLine() splits string at `\n`, `\r`, or `\r\n` and removes newline characters.
Python's readline() works in the same way at universal newline mode (default). but replaces newline characters to `\n`. So we don't need to remove `\r`, I guess.

**Comment by hiroshi-matsuda** [comment]
I guess, too. Thanks a lot!
I'd like to remove \r from rstrip argument.
And then, may I merge this branch?

**Pull request merged**